### PR TITLE
fix deflate decompression for aiohttp

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by xiangyan99** [comment]
#28477

File: sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ async def __anext__(self):
                 if not self._decompressor:
                     import zlib
 
-                    zlib_mode = 16 + zlib.MAX_WBITS if enc == "gzip" else zlib.MAX_WBITS
+                    zlib_mode = 16 + zlib.MAX_WBITS if enc == "gzip" else -zlib.MAX_WBITS
```
Comment (iscai-msft) [inline sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py:335]: is the `else` 16 - zlib.MAX_WBITS` or just `-zlib.MAX_WBITS`. I think it's the latter but it's not the clearest

**Review by iscai-msft — APPROVED** [review]

File: sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py (hunk 1 above)
Comment (xiangyan99) [reply sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py:335]: It is just -zlib.MAX_WBITS

File: sdk/core/azure-core/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
+## 1.26.3 (2023-02-02)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Fixed deflate decompression for aiohttp   #28483
```
Comment (kashifkhan) [inline sdk/core/azure-core/CHANGELOG.md:5]: nit: will the heading for this be bug fix ?

**Review by kashifkhan — APPROVED** [review]

File: sdk/core/azure-core/CHANGELOG.md (hunk 2 above)
Comment (xiangyan99) [reply sdk/core/azure-core/CHANGELOG.md:7]: Good catch. It was an accident. :)

**Pull request merged**